Automattic/jetpack
# Publicize: Rename Publicize to Jetpack Social

This PR replaces all user-facing instances of Publicize with Jetpack Social.

#### Changes proposed in this Pull Request:

* Replace Publicize with Jetpack Social.
* If applicable, replace Publicize with "share" (when Publicize is used as a verb).

#### Other information:


#### Jetpack product discussion


pdrWKz-vt-p2

#### Does this pull request change what data or activity we track or use?


No.

#### Testing instructions:






* Make sure tests still pass.
* In general, since they're just textual changes, go through the code and make sure nothing has changed beyond just textual changes.
* Make sure the name is updated in the visible places:

<img width="1039" alt="image" src="https://user-images.githubusercontent.com/1713699/186190884-70097cd0-70f7-42fd-ad50-efe5c095d225.png">


<img width="305" alt="image" src="https://user-images.githubusercontent.com/1713699/186183443-31eb93d2-b2cc-4011-a327-928e8d9d8bff.png">
